script.js

```diff
@@ -155,6 +154,59 @@ async function main () {
   }
 }
 
+/*
+ * ==============================================================
+ *                         BUILD
+ * ==============================================================
+ */
+
+const tempHtmlPath = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
```

Comment:
Hey can you change this
```suggestion
const tempHTMLpath = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
```